lib/main.js

```diff
@@ -1,9 +1,13 @@
-var _ = require('lodash');
-var css = require('css');
+'use strict';
+
 var cheerio = require('cheerio');
+var postcss = require('postcss');
+var cssWhat = require('css-what');
 var Hashids = require('hashids');
 
-var ALPHABET = 'abcdefghijklmnopqrstuvwxyz';
+var stringify = require('./css-what/stringify');
+
+var ALPHABET = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
```

Comment:
Does html read  `"somID"` differently than `"someid"`. I think it's case insensitive - so having the upper case could be dangerous here?